### PR TITLE
add manpages for welle-io & welle-cli

### PR DESCRIPTION
**Pull request by tenzap**

### CODE REVIEW

File: src/welle-cli/doc/man/update_manpage.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
Comment (herrhotzenplotz) [inline src/welle-cli/doc/man/update_manpage.sh:1]: Nit: Unportable. Use `/bin/sh` instead.

File: src/welle-gui/doc/man/update_manpage.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (herrhotzenplotz) [inline src/welle-gui/doc/man/update_manpage.sh:1]: Same as above. This is unportable. Use `/bin/sh`instead.

**Review by herrhotzenplotz — COMMENTED** [review]
Just something that jumped into my eye.

**Review by AlbrechtL — CHANGES_REQUESTED** [review]
@herrhotzenplotz Thanks for checking!

@tenzap Please update the update scripts like @herrhotzenplotz recommended

**Review by alexmyczko — APPROVED** [review]

**Comment by alexmyczko** [comment]
i had prepared manpages i forgot to send: https://sources.debian.org/src/welle.io/2.2-1/debian/

**Comment by AlbrechtL** [comment]
@alexmyczko Stupid question: Do have I something to do with your man pages? As far as I see this PR includes man pages.

**Comment by alexmyczko** [comment]
@AlbrechtL no, unless you feel like comparing the PR ones with the ones i have, and taking the best parts for the final version... but you can absolutely skip this step

**Pull request merged**